src/sentry/search/events/fields.py

```diff
@@ -2420,6 +2436,9 @@ def __init__(self, dataset: Dataset, params: ParamsType):
                 ),
             ]
         }
+        # In Performance TPM is used as an alias to EPM
```

Comment:
nit: I don't think this comment makes sense here, could we also move `FUNCTION_ALIASES` to search/events/constants?